src/main/kotlin/no/nav/tilleggsstonader/sak/behandlingsflyt/task/OpprettOppgaveForOpprettetBehandlingTask.kt

```diff
@@ -88,7 +88,9 @@ class OpprettOppgaveForOpprettetBehandlingTask(
                 type = TYPE,
                 payload = objectMapper.writeValueAsString(data),
                 properties = Properties().apply {
-                    this["saksbehandler"] = data.saksbehandler
+                    if (data.saksbehandler != null) {
+                        this["saksbehandler"] = data.saksbehandler
+                    }
                     this["behandlingId"] = data.behandlingId.toString()
```

Comment:
Ikke koblet til selve PRen men vi burde nog bruke `setProperty` i stedet for `this["~"]`